docs/data_loaders_on_gpu_with_jax.md

```diff
@@ -0,0 +1,678 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.15.2
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
++++ {"id": "PUFGZggH49zp"}
+
+## Introduction to Data Loaders on GPU with JAX
+
++++ {"id": "3ia4PKEV5Dr8"}
+
```

Comment:
Presumably this will come right after data loaders on CPU... can we add an overview paragraph saying what is different in this context? I just skimmed the two, and it's not obvious to me.

Reply:
Any response here? Thanks!